test/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -271,6 +283,33 @@ import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 				done();
 			});
 		});
+		test('updating filter option updates filters', function() {
+			const update = 2;
+			_setSelectedOptions([0, 4, 5]);
+			const expected = _getExpectedAndImport(filter);
+			filter.updateFilterOptionSelectedStatus(options[update].cat, options[update].key, true);
+			_updateExpectedOption(update, true, expected);
+			assert.deepEqual(expected, filter._filters);
+		});
+		test('updating filter option updates display', function(done) {
+			const update = 2;
+			const selected = [0, 4, 5];
+			_setSelectedOptions(selected);
+			const expected = _getExpectedAndImport(filter);
+			filter.updateFilterOptionSelectedStatus(options[update].cat, options[update].key, true);
```

Comment:
Again, I don't see where `options` is defined. If it's a file level private variable, should we prefix it with an underscore? e.g. `_options`

Reply:
Never mind this - was addressed above, and I don't think that we need the underscore prefix